### PR TITLE
Networking and MySQL handling improvements and fixes

### PR DESCRIPTION
**Pull request by bvitnik**

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -234,6 +259,8 @@ deploy_zone() {
   cmk set asyncblock true
   cmk sync
 
+  echo bla
```
Comment (taocnrut) [inline install.sh]: i love this part :)

Comment (bvitnik) [reply install.sh]: That's the hidden feature 😁

Comment (yadvr) [inline install.sh]:
```suggestion
  info "Starting Zone deployment"
```

**Review by yadvr — APPROVED** [review]
LGTM, fantastic & thanks @bvitnik for the pull request!

**Pull request merged**